### PR TITLE
Adds Pull Request template #4

### LINKED ISSUE #4: Add Pull Request Template
Add pull request template for this repository. For more information on how to do this refer [this](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/).

### PR DESCRIPTION
**Pull request by manigedit**
Adds pull request template for the repository

Description
[*] Feature

Fixes #4

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```
Comment (pradeepgangwar) [inline .github/pull_request_template.md]: Write this line as comment.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+Fixes # (issue)
```
Comment (pradeepgangwar) [inline .github/pull_request_template.md]: Add a new line between above two lines.

**Comment by pradeepgangwar** [comment]
@manigedit In your PR also mention which issue it solves. Like this : `Fixes #<issue_no.>`

**Comment by manigedit** [comment]
Please review now

File: .github/pull_request_template.md (hunk 1 above)
Comment (pradeepgangwar) [reply .github/pull_request_template.md]: @manigedit It is not yet fixed. How can u mark it as resolved?

File: .github/pull_request_template.md (hunk 2 above)
Comment (pradeepgangwar) [reply .github/pull_request_template.md]: @manigedit Please add a blank line as said.

**Comment by pradeepgangwar** [comment]
@manigedit U didn't resolve issues that i mentioned in inline comments. You marked them resolved without fixing :/ . Please fix.

**Comment by pradeepgangwar** [comment]
@manigedit Any updates? Else we will close the PR to allow others to work on this.

**Comment by manigedit** [comment]
@Tlazypanda  sorry for delay, I'm fixing it now.

**Comment by pradeepgangwar** [comment]
Closing this as @manigedit Opened a new PR for this.

**Pull request closed without merging**